### DISCUSSION TITLE
trimming model for single language other than english?

### DISCUSSION THREAD

**Discussion by alejandro-anv** [openai/whisper#723] [Q&A] [answered]
is there a way of getting a single language model other than english for reducing the memory usage? my card is only 5Gb and according to the documentation it's not enough for a large multilingual model. 

Is there any way of trimming a model file for removing unnecesary languages? (I will need spanish only)

**Comment by krylm** [comment, marked as answer]
Maybe try model finetuned specially for your language with smaller size (medium).

https://huggingface.co/juancopi81/whisper-medium-es
You can try it online if it will be good enough for your usage.
https://huggingface.co/spaces/juancopi81/whisper-demo-es-medium
